### ISSUE TITLE
Doesn't work on alfred v4.0.1

### ISSUE THREAD

**Issue by bryanvlierop** [yannickglt/alfred-devdocs#32]
The workflow doesn't work for alfred v4.0.2 

After typing doc alfred continues to google the next arguments.

<img width="545" alt="Screenshot 2019-06-17 at 19 33 47" src="https://user-images.githubusercontent.com/18574094/59624378-e3f11580-9136-11e9-82b2-35453b2ae090.png">

**Comment by the21st** [comment]
Same here

**Comment by Firxiao** [comment]
Here is a workaround. Please take a try. Notice: only for 4.x
```bash
curl https://devdocs.io/docs/docs.json -o "$HOME/Library/Caches/com.runningwithcrayons.Alfred/Workflow Data/com.yannickglt.alfred2.devdocs/docs.json"
```

**Comment by Mr-jiangzhiguo** [comment]
same

**Comment by yannickglt** [comment]
Should be fixed in version 4.1.0 of the workflow

**Comment by rkoval** [comment]
@yannickglt where can 4.1.0 be downloaded? Packal doesn't appear to have a valid workflow for Alfred 4, and neither do the GitHub releases for this repository. I've tried cloning the git repo on master and just using the `src` as-is in my Alfred workflows directory, but I think there might be some bad global state/cache somewhere that's still preventing use of this workflow.
